resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -110,6 +110,8 @@ aws.terminal.action=Open AWS Local Terminal
 aws.terminal.action.tooltip=Start a local terminal with the current AWS connection settings injected as environment variables
 aws.terminal.exception.failed_to_resolve_credentials=Unable to open AWS-aware Terminal, unable to resolve credentials: {0}
 aws.terminal.exception.invalid_credentials=Unable to open AWS-aware Terminal, credentials aren't valid: {0}
+aws.toolkit_deprecation.title=AWS Toolkit deprecation notice
+aws.toolkit_deprecation.message=Support for {0} {1} is being deprecated - an upcoming release will require a version based on {2} or newer.
```

Comment:
I reckon you could do:

`Support for {0} {1} is being deprecated - an upcoming release will require {0} {2} or newer.`

Which (presumably) would translate into :

"Support for IntelliJ IDEA 2020.1 is being deprecated - an upcoming release will require IntelliJ IDEA 2020.2 or newer."

Reply:
but whatever - we're nitpicking at this point - the wording looks good.